rulebook.tex

```diff
@@ -2025,34 +2058,46 @@ \section{Hardware Specification}
 \subsection{Robot Dimensions}
 \begin{figure}
   \includegraphics{figures/dimensions.pdf}
-  \caption{Robot dimensional constraints (top down view).}
+  \caption{Robot dimensional constraints (top down view), derived from the Robotino platform.}% chktex ignore-long-line
   \label{fig:dimensions}
 \end{figure}
 
+\subsection{Physical Constraints}
+The Festo Robotino\footnote{\href{https://robotino.com}{robotino.com}} by
+Festo Didactics is the reference platform for the \ac{RCLL}. As such,
+we derive specific limitations and constraints from the Robotino platform.
+The Robotino is a circular, omnidirectional robot with a diameter of
+\SI{45}{\centi\metre}. It is recommended to use omnidirectional drive systems
+and a circular base for the robot. The robot must be able to withstand
+non-deliberate impacts of other robots or obstacles (e.g., machines).
+
 The robots dimensions (including sensors, computing equipment, etc.) must be
 within a cylindrical bounding box with maximum total height of \SI{1.6}{\metre}
 (relative to the ground) and a diameter of
 \SI{0.55}{\metre} (centered at the robot's rotational
 center-point, green \emph{Robotino base area} plus blue \emph{Extended area} of
 \reffig{fig:dimensions}) during regular operation.
 While  in front of a machine and during a production process, the gripper may
-be extended up to \SI{30}{\centi\metre} beyond the bounding box
+be extended up to \SI{30}{\centi\metre} beyond this bounding box
 (red \emph{Actuator reach} of \reffig{fig:dimensions}).
 But it is only allowed to reach a maximum of
 \SI{15}{\centi\metre} into the machine area.
 
+All robots are restricted to
+operate with a maximum speed of \SI[fraction-function=\dfrac]{1}{\metre/\second}
```

Comment:
we should re-measure i am pretty sure the actual top speed of the robotino is betwen 0.5 and 0.7 m/s

Reply:
it's 0.7m/s